src/wroboclaw/roboclaw/serial_interface.py

```diff
@@ -198,6 +198,12 @@ def get_over_current_status(self) -> Tuple[Optional[bool], Optional[bool]]:
             return self._curr_l > self._curr_lim_l if self._curr_lim_l is not None else None, \
                 self._curr_r > self._curr_lim_r if self._curr_lim_r is not None else None
 
+    def set_counts_per_rotation(self, counts_per_rotation_l : int = None, counts_per_rotation_r : int = None) -> None:
+        pass
+
+    def set_offset(self, offset_l : int = None, offset_r : int = None) -> None:
+        pass
+
```

Comment:
Maybe comment to signal why these methods are do-nothings.

Reply:
Imagine someone else coming by in 2-3 years...
"These methods don't have any behavior; I'll just remove them!"
And the woe that befalls them afterwards.

Reply:
Complete as of cb491a0.